app/static/js/volunteerProfile.js

```diff
@@ -0,0 +1,71 @@
+$(document).ready(function(){
+
+  $(".form-check-input").click(function updateInterest(){
+    var programID = $(this).attr('id');
+    var interest = $(this).is(':checked');
+    var username = $(this).attr('name');
+
+    var routeUrl = interest ? "addInterest" : "deleteInterest;

```

Comment:
syntax error

Reply:
Fixed!

Reply:
Noticed and fixed with a previous commit